routers/web/repo/view.go

```diff
@@ -1095,7 +1095,10 @@ func checkOutdatedBranch(ctx *context.Context) {
 	}
 
 	if dbBranch.CommitID != commit.ID.String() {
-		ctx.Flash.Warning(ctx.Tr("repo.error.broken_git_hook"), true)
+		ctx.Flash.Warning(ctx.Tr(
+			"repo.error.broken_git_hook",
+			"https://docs.gitea.com/help/faq#push-hook--webhook--actions-arent-running",
+		), true)
```

Comment:
```suggestion
		ctx.Flash.Warning(ctx.Tr("repo.error.broken_git_hook", "https://docs.gitea.com/help/faq#push-hook--webhook--actions-arent-running"), true)
```

Could we keep it one line? Sooner or later, I think we might need to use regexp to filer the translation keys .... there are too many legacy problem in the locale system.